docs/conceptual_guides/quantization_schemes.md

```diff
@@ -0,0 +1,88 @@
+```
+
+where:
+
+- `x_q` is the quantized `int8` value associated to `x`
+- `S` is the scale, and is a positive `float16`
+- `Z` is called the zero-point, it is the `int8` value corresponding to the value 0 in the `float16` realm. If zero-point is ommited, we call this "symmetric" quantization.
+
+
+The quantized value x_q of x in [a, b] can be computed as follows:
+
+```bash
+x_q = round(x/S + Z)
+```
+
+And `float16` values outside of the [a, b] range are clipped to the closest representable value, so for any floating-point number x:
+
+```bash
+x_q = clip(round(x/S + Z), round(a/S + Z), round(b/S + Z))
+```
+
+## Quantization Flavors
+
+There are several flavors of quantization.
+
+### Static vs Dynamic
+
+The section above described how quantization from `float16` to `int8` works, but did not explain how to compute the scales and zero points.
+
+With weights, since the full range is known ahead of time, we can just compute the scales and zero points statically (sometimes using a more sophisticated algorithm like `GPTQ`).
+
+With activations, however, there are two approaches:
+* **Dynamic quantization**: the range for each activation is computed on the fly at runtime. This gives us the best possible values, but it can be a bit slower than static quantization because of the overhead introduced by computing the range each time. It is also not an option on certain hardware.
+
+* **Static quantization**: the range for each activation is computed in advance at quantization-time, typically by passing representative "calibration" data through the model and recording the activation values. In practice, we run a number of forward passes on a calibration dataset is done and compute the ranges according to the observed calibration data.
+
+In general, it is best practice to start your experiments with:
+- For `fp8`, use static activation quantization
+- For `int8`, use dynamic activation quantization
+
+### Granularity
+
+Weight and activation quantization can be performed at different levels of granularity depending on accuracy / latency tradeoffs being targeted.
+
+#### Weight Quantization Granularity
+
+For weight quantization, there are three "levels" (in order of increasing of granularity):
+* **Per-Tensor**: one pair of quantization parameters (`S, Z`) is used per tensor
+* **Per-Channel**: one pair of quantization parameters (`S, Z`) is used per element of one of the dimensions of the tensor. For instance, with a weight matrix of shape `[N,M]`, the scales are a vector of shape [`M`] scales.
+* **Per-Group**: one pait of quantization parameters is (`S, Z`) is used per group of items in a tensor. For instance, with a weight matrix of shape `[N,M]` with `M=4096`, the scales are a matrix of shape `[32, M]` (note: `4096 / 128 = 32`).
+
+Incresing quantization granularity typically helps with accuracy at the expense of less memory reduction and slower inference performance. In general, it is best practice to start your experiments with:
```

Comment:
```suggestion
Incresing quantization granularity typically helps with accuracy at the expense of less memory reduction and slower inference performance.  This is because we compute quantization ranges over smaller distributions with the trade off of needing more memory to represent them. In general, it is best practice to start your experiments with:
```